doc/BUILDSERVER.md

```diff
@@ -0,0 +1,160 @@
+
+This setting can be modified on the respective build job configuration site.
+
+## Triggers
+All Triggers are described in the configuration of the respective build jobs.
+
+The
+[daily build](https://build.libelektra.org/jenkins/job/elektra-jenkinsfile-daily/)
+is executed according to a cron schedule.
+
+The [libelektra](https://build.libelektra.org/jenkins/job/libelektra/)
+build is triggered for all branches of the libelektra repository except for
+`debian`.
+Additionally all open branches in forks targeting libelektra's repository via
+PRs are going to be build.
+Pushes to any of those branches will trigger a new build automatically.
+
+The following phrases can be used as comments to manually trigger a specific
+build:
+
+* jenkins build [gcc-configure-debian](https://build.libelektra.org/job/elektra-gcc-configure-debian/) please
+* jenkins build [gcc-configure-debian-debug](https://build.libelektra.org/job/elektra-gcc-configure-debian-debug) please
+* jenkins build [gcc-configure-debian-intree](https://build.libelektra.org/job/elektra-gcc-configure-debian-intree/) please
+* jenkins build [gcc-configure-debian-musl](https://build.libelektra.org/job/elektra-gcc-configure-debian-musl/) please
+* jenkins build [gcc-configure-debian-nokdbtest](https://build.libelektra.org/job/elektra-gcc-configure-debian-nokdbtest/) please
+* jenkins build [gcc-configure-debian-notest](https://build.libelektra.org/job/elektra-gcc-configure-debian-notest/) please
+* jenkins build [gcc-configure-debian-shared](https://build.libelektra.org/job/elektra-gcc-configure-debian-shared/) please
+* jenkins build [gcc-configure-debian-optimizations](https://build.libelektra.org/job/elektra-gcc-configure-debian-optimizations/) please
+* jenkins build [gcc-configure-debian-withspace](https://build.libelektra.org/job/elektra-gcc-configure-debian-withspace/) please
+* jenkins build [gcc-configure-xdg](https://build.libelektra.org/job/elektra-gcc-configure-xdg/) please
+* jenkins build [gcc-i386](https://build.libelektra.org/job/elektra-gcc-i386/) please
+* jenkins build [git-buildpackage-jessie](https://build.libelektra.org/job/elektra-git-buildpackage-jessie/) please
+* jenkins build [git-buildpackage-wheezy](https://build.libelektra.org/job/elektra-git-buildpackage-wheezy/) please
+* jenkins build [icc](https://build.libelektra.org/job/elektra-icc/) please
+* jenkins build [ini](https://build.libelektra.org/job/elektra-ini-mergerequests/) please
+* jenkins build [local-installation](https://build.libelektra.org/job/elektra-local-installation/) please
+* jenkins build [mingw64](https://build.libelektra.org/job/elektra-gcc-configure-mingw-w64/) please
+* jenkins build [multiconfig-gcc-stable](https://build.libelektra.org/job/elektra-multiconfig-gcc-stable/) please
+* jenkins build [source-package-test](https://build.libelektra.org/job/elektra-source-package-test/) please
+* jenkins build [libelektra](https://build.libelektra.org/jenkins/job/libelektra/) please
```

Comment:
This job first? Explain its relevance?

Reply:
No relevance at all. Just added to the bottom of 'older' triggers. Should I insert them alphabetically?

Reply:
Stupid question: Why not renaming the jobs from "libelektra" and "libelektra-daily" to "elektra" and "elektra-daily"? We already has similar discussions for the repo #1778. An advantage would be that it would be first in alphabetical order (and hence in the list of the trigger phrases). Not really important, though. (we can also order by "importance").